src/components/pagination/Pagination.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import styles from './pagination.module.scss'; 
+import Right from '../../assets/images/icon-caret-right.svg?react';
+import Left from '../../assets/images/icon-caret-left.svg?react';
+
+// TODO: functionalitate butoane
+export default function Pagination () {
+    const [viewportWidth, setViewportWidth] = useState(window.innerWidth); 
+    
+    window.addEventListener('resize', () => {
+        setViewportWidth(window.innerWidth);
+    })
+
+    return(
+        <div className={styles.pagination}>
```

Comment:
You'll hate me, but I don't like the way css modules are supposed to be put to use, so I think your initial proposal has a better developer experience. If you have the time, revert it. 🙈 🙉 🙊

Reply:
It was a good exercise, I learned how css module work 😀